validation/process_rlimits.go

```diff
@@ -17,7 +17,12 @@ func main() {
 	if err != nil {
 		util.Fatal(err)
 	}
-	g.AddProcessRlimits("RLIMIT_NOFILE", 1024, 1024)
```

Comment:
`RLIMIT_NOFILE` is also [in POSIX][1], so I don't think we want to drop this one.  I'm in favor of adding the others.

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/functions/getrlimit.html

Reply:
It has been added by [GetDefaultGenerator()](https://github.com/opencontainers/runtime-tools/blame/master/generate/generate.go#L70).

Reply:
However the default generator uses the same value 1024 for the soft and hard value.

Reply:
updated, I add `RLIMIT_NOFILE` and use different value.